build.gradle

```diff
@@ -400,6 +396,17 @@ uploadArchives {
                 ' for more information.' )
             throw new Exception("Missing sonatype username or password, please see the README for properties that must be specified for uploading an archive.")}
         )
-
     }
+
+distributions {
+    spark{} //added to allow an installSparkDist target which will also build the spark jar
+}
+
+installSparkDist.dependsOn downloadGsaLibFile, sparkJar
```

Comment:
`installSparkDist`? This is not quite the easy-to-remember name I was hoping for for the "build everything" target. How about: `gradle installDist` behaves as it does currently, we rename `gradle sparkJar` to `gradle installSpark`, and we add a `gradle installAll` that does both? That way, we have `gradle installDist`, `gradle installSpark`, and `gradle installAll`, which is at least consistent naming-wise. Thoughts from others on the team?